### ISSUE TITLE
NTR: 4-hydroxyphenyllactic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8575]
New term request 4-hydroxyphenyllactic acid high in urine (segal_030509152427): P-hydroxyphenyllactate  UMLS: 1848679

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000275
- label: 4-hydroxyphenyllactic aciduria
- parents: [Elevated urinary carboxylic acid (HP:0040156)](https://hpo.jax.org/app/browse/term/HP:0040156)
- pmid: 
- def: Amount of 4-hydroxyphenyllactic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8575
- synonyms:
